pysat/utils/files.py

```diff
@@ -122,13 +127,13 @@ def process_parsed_filenames(stored, two_digit_year_break=None):
 
 
 def parse_fixed_width_filenames(files, format_str):
-    """Parse list of files, extracting data identified by `format_str`.
+    """Extract time/version info from a list of files with a fixed name width.
```

Comment:
```suggestion
    """Extract time/version/custom info from a list of files with a fixed name width.
```
Thanks for the improvement. If we can fit in that we parse out all provided keywords it would be even better.

Reply:
```suggestion
    """Extract specific info from a list of files with a fixed name width.
```